content/docs/command-reference/plots/index.md

```diff
@@ -96,7 +84,143 @@ names in the `train` array below:
+`plots` key.
+
+```yaml
+# dvc.yaml
+stages: ...
+
+plots: ...
+```
+
+Every plots has to have its own id. Configuration, if provided, should be a
+dictionary.
+
+In simplest use case, user can provide file path as the plot id and don't
+provide configuration at all:
+
+```yaml
+# dvc.yaml
+---
+plots:
+  logs.csv:
+```
+
+In that case the default behavior will be applied. DVC will take data from
+`logs.csv` file and apply `linear` plot
+[template](/doc/command-reference/plots#plot-templates) to the last found column
+(CSV, TSV files) or field (JSON, YAML).
+
+We can customize the plot by adding appropriate fields to the configuration:
+
+```yaml
+# dvc.yaml
+---
+plots:
+  confusion_matrix:
+    y:
+      confusion_matrix_data.csv: predicted_class
+    x: actual_class
+    template: confusion
+```
+
+In this case we provided `confusion_matrix` as a plot id. It will be displayed
+in the plot as a title, unless we override it with `title` field. In this case
+we provided data source in `y` axis definition. Data will be sourced from
+`confusion_matrix_data.csv`. As `y` axis we will use `predicted_class` field. On
+`x` axis we will have `actual_class` field. Note that DVC will assume that
+`actual_class` is inside `confusion_matrix_data.csv`.
+
+We can provide multiple columns/fields from the same file:
+
+```yaml
+#dvc.yaml
+---
+plots:
+  multiple_series:
+    y:
+      logs.csv: [accuracy, loss]
+    x: epoch
+```
+
+In this case, we will take `accuracy` and `loss` fields and display them agains
+`epoch` column, all coming from `logs.csv` file.
+
+We can source the data from multiple files too:
+
+```yaml
+#dvc.yaml
+---
+plots:
+  multiple_files:
+    y:
+      train_logs.csv: accuracy
+      test_logs.csv: accuracy
+    x: epoch
+```
+
+In this case we will plot `accuracy` field from both `train_logs.csv` and
+`test_logs.csv` against the `epoch`. Note that both files have to have `epoch`
+field.
+
+#### Avialable configuration fields
+
+- `x` - field name from which the X axis data comes from. An auto-generated
+  _step_ field is used by default. It has to be a string.
+
+- `y` - field name from which the Y axis data comes from. It can be a string,
+  list or dictionary. If its a string or list, it is assumed that plot id will
+  be the path to the data source. String, or list elements will be the names of
+  data columns or fields withing the source file. If this field is a dictionary,
+  it is assumed that its keys are paths to data sources. The values have to be
+  either strings or lists, and are treted as column(s)/field(s) within
+  respective files.
+- `x_label` - X axis label. The X field name is the default.
+- `y_label` - Y axis label. If all provided Y entries have the same field name,
+  this name will be the default, `y` string otherwise.
+- `title` - Plot title. Plot id by default.
+
+Refer to the
+[examples](/doc/command-reference/plots#example-simple-plot-definition) for more
+syntax insight.
+
+### Stage plots
```

Comment:
WDYT about moving this above Syntax and then mentioning in Syntax when some options are specific to standalone plots definitions?

Reply:
Syntax is still part of plot configuration, so I would go with h3, to make it under `Configuring a plot`